contracts/winnerApproval.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract WinnerApproval is Ownable {
+    /// Winner Vote should require three unique people to have to verify
+    /// that you completed the program before you're accepted.
+    /// All of these people have to be past winners.
+    /// Once you're accepted, then you can mint the token
+
+    uint private numApprovalsRequired;
+    IERC20 private token;
+    uint private minNumTokens;
+
+    event WinnerVerified(address indexed _user, bool _byOwner, bool _byApproval);
+    event ApprovalGranted(address indexed _approver, address _approvee);
+
+    struct winnerDetails {
+        bool winner;
+        address[] winnerApprovals;
+    }
+    mapping(address => winnerDetails) private winners;
```

Comment:
```suggestion
    struct winnerDetails {
        bool winner;
        address[] winnerApprovals;
    }
    mapping(address => winnerDetails) private winners;
```

IMO can be improved:
    struct player {
        bool hasWon;
        address[] judgesWhoApproved;
    }